api/crossmodelrelations/crossmodelrelations.go

```diff
@@ -76,9 +76,9 @@ func (c *Client) RegisterRemoteRelations(relations ...params.RegisterRemoteRelat
 }
 
 // WatchRelationUnits returns a watcher that notifies of changes to the
-// units in the remote model for the relation with the given remote id.
-func (c *Client) WatchRelationUnits(remoteRelationArg params.RemoteRelationArg) (watcher.RelationUnitsWatcher, error) {
-	args := params.RemoteRelationArgs{Args: []params.RemoteRelationArg{remoteRelationArg}}
+// units in the remote model for the relation with the given remote token.
+func (c *Client) WatchRelationUnits(remoteRelationArg params.RemoteRelationDetails) (watcher.RelationUnitsWatcher, error) {
```

Comment:
I find it surprising that a "details" struct should serve as a parameter struct. I think it would be best to create a separate type for the params. They may have the same contents initially, but this will allow them to diverge naturally.